website/docs/r/network_acl.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_network_acl"
+sidebar_current: "docs-alicloud-resource-network-acl"
+description: |-
+  Provides a Alicloud Network Acl resource.
+---
+
+# alicloud\_network_acl
+
+Provides a network acl resource to add network acls.
+
+-> **NOTE:** Terraform will auto build network acl instance while it uses `alicloud_network_acl` to build a network acl resource.
+
```

Comment:
Please add a note: `-> **NOTE:** Available in 1.43.0. Currently, the resource are only available in Hongkong(cn-hongkong), India(ap-south-1), and Indonesia(ap-southeast-1) regions.`.